package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "developercenter-tutorials",
+  "version": "1.0.0",
+  "description": "This repo is pulled into the [PureCloud Developer Center](http://developer.mypurecloud.com) and contains walkthroughs of applications and examples.  \r This project is based off of [Viewsaurus](https://www.npmjs.com/package/viewsaurus) which is MIT licensed at version 0.3.1 but adds multi language support.",
+  "main": "index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jenissabarrera/developercenter-tutorials.git"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Use `MIT` for the license

Reply:
I just realized this package.json was in the root of the repo. Can you remove this and the package-lock.json file? The root of the repo shouldn't be a node package, just the individual tutorials.

Reply:
Hi Tim. We already updated this repository. You may now check the updated pull request for the updates. Thanks.

Reply:
This PR still contains package.json and package-lock.json in the root of the repo. Please remove them.

Reply:
files are already removed.